server/app/durablejobs/AbstractDurableJobRunner.java

```diff
@@ -55,7 +55,8 @@ public AbstractDurableJobRunner(
     this.durableJobRegistry = Preconditions.checkNotNull(durableJobRegistry);
     this.itEmailAddress =
         config.getString("it_email_address").isBlank()
-            ? config.getString("support_email_address")
+            // TODO(#11736): Set support_email_address as the fallback email
+            ? "civiform-dev@exygy.com"
```

Comment:
This email is used when a durable job fails, so it's fine for it to come to us. This is the value that was being used as a fallback anyway since we can't pick up the value they set in the config.